server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileBackuper.java

```diff
@@ -52,29 +51,33 @@
 @ServerSide
 public class QProfileBackuper {
 
+  private static final Joiner RULE_KEY_JOINER = Joiner.on(", ").skipNulls();
+
   private final QProfileReset reset;
   private final DbClient db;
 
-  private static final Joiner RULEKEY_JOINER = Joiner.on(", ").skipNulls();
-
   public QProfileBackuper(QProfileReset reset, DbClient db) {
     this.reset = reset;
     this.db = db;
   }
 
+  /**
+   * @deprecated use {@link #backup(DbSession, QualityProfileDto, Writer)} instead
+   */
+  @Deprecated
   public void backup(String key, Writer writer) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)